### DISCUSSION TITLE
Consider of reverting renaming Arduino to Sloeber

### DISCUSSION THREAD

**Discussion by ArminJo** [Sloeber/arduino-eclipse-plugin#1439] [Ideas]
Hi Jaintje,
I recently upgraded to the latest nightly, and discovered, that you renamed Arduino to Sloeber at different places in the GUI.
At first, I thought the project properties menu was broken. Arduino was quite at the top of the Menu, while Sloeber is buried way down. And it contains the Arduino specific settings for Eclipse.
I can imagine, that the name Sloeber was a bit underrepresented in the GUI, or do you got trademark problems?
Personally I would prefer the old (Arduino instead of Sloeber) style, it is also better to teach.

And another reason is, that I have to update all of my screenshots in the [intro tutorial](https://www.dropbox.com/s/qk6gqtyialzmmha/Sloeber_Intro.pdf?dl=0).
BTW, are you interested in any parts of the tutorial for the online documentation?

Best regards
Armin

**Comment by jantje** [comment]
I am not considering to consider reverting to arduino because I have been thinking about this problem for a long time and finally decided to bit the bullet.
I understand this is not a popular decision with the old Sloeber users, but there was confusion because there are other eclipse plugins that use the name Arduino (some still in CDT-and thus in sloeber-) 
The plugin used to be called "the arduino eclipse plugin" but legallese (from eclipse not Arduino) forced us to change the name.
At that time we voted for sloeber (Software Library for Open Embedded Boards on a Eclipse Runtime) -The name of my cat featuring in the beginning of my Sloeber videos that just died-. We changed many references to arduino to sloeber but not all. 
Now I think we use "sloeber" for everything related to the eclipse plugin and "arduino" for everything Arduino related (boards,platforms,libraries)

>At first, I thought the project properties menu was broken. Arduino was quite at the top of the Menu, while Sloeber is buried way down.

I noticed that one to. That is why I added a button to the toolbar that sends you straight to the sloeber settings.

> And it contains the Arduino specific settings for Eclipse.

It not only contains arduino specific settings anymore as now it also contains the version control settings in the other tab.

>I can imagine, that the name Sloeber was a bit underrepresented in the GUI,

I don't care about that.

>or do you got trademark problems?

We did a long time ago as explained above. The main reason is to avoid confusion with other plugins and dead -not functioning code- that make people think sloeber doesn't work. And I'm autistic and I can't handle "close but no sigar"

>Personally I would prefer the old (Arduino instead of Sloeber) style, it is also better to teach.

Everybody has his preferences. I'm not sure it is better to teach as one should make a difference between what sloeber does, what other plugins do and what arduino does.

>And another reason is, that I have to update all of my screenshots in the intro tutorial.

I know, I should do that to. 

>BTW, are you interested in any parts of the tutorial for the online documentation?

I thought I already linked to it somewhere. We should also put a link to it on the intro page
https://github.com/Sloeber/arduino-eclipse-plugin/blob/master/website/WebContent/product/V4_0.php
Maybe put a release of the doc in the github repository as well

**Reply by ArminJo** [reply]
Thanks a lot for the detailed answer. 👍 

> I noticed that one to. That is why I added a button to the toolbar that sends you straight to the sloeber settings.

This does not lead to the project settings 😞. But I will get used to 😀.

> Maybe put a release of the doc in the github repository as well

[Here is the editable word version](https://www.dropbox.com/s/866dvw5tcb9jlny/Sloeber_Intro.docx?dl=0)

> We did a long time ago as explained above. The main reason is to avoid confusion with other plugins and dead -not functioning code- that make people think sloeber doesn't work. And I'm autistic and I can't handle "close but no sigar"

I understand! I did the same with the IRremote library with using *.hpp files.

Thanks and best regards
Armin

**Comment by jantje** [comment]
>This does not lead to the project settings 😞. But I will get used to 😀.

I have been in a remote debugging session (read video cam with someone else behind the buttons/wires) all day with a freshly installed sloeber from the actions and we used this feature at least 10 times today. So this should work.
There is a button for the project properties and one for the preferences.  I just made a video about it and I saw the caption states preferences for both.
I will link to the video when I uploaded it.

**Reply by ArminJo** [reply]
Ok, I missed the new button. Great idea!!! Thanks for it.

**Reply by jantje** [reply]
Here is the video https://youtu.be/ApBw-z9vjKY
